db/migrate/20170317155628_add_details_to_splits.rb

```diff
@@ -0,0 +1,8 @@
+class AddDetailsToSplits < ActiveRecord::Migration
+  def change
+    add_column :splits, :hypothesis, :text, null: true
+    add_column :splits, :assignment_criteria, :text, null: true
+    add_column :splits, :description, :text, null: true
+    add_column :splits, :squad_owner, :string, null: true
```

Comment:
Also, TestTrack already has a large glossary of terms surrounding it, so i'm not super psyched about adding another term -- "Squad" -- to the list. I'm fine with being overruled and we just add the idea of squads as a unit that owns splits to the readme somewhere. We just don't want to lose focus of the fact that this is a product that we should strive to keep as simple as humanly possible. ❤️